src/ui/modal/modalConfirmThumbnailUpload/index.js

```diff
@@ -1,11 +1,12 @@
 import { connect } from 'react-redux';
 import { doHideModal } from 'redux/actions/app';
 import { doUploadThumbnail, doUpdatePublishForm } from 'lbry-redux';
+import fs from 'fs';
 import ModalConfirmThumbnailUpload from './view';
 
 const perform = dispatch => ({
   closeModal: () => dispatch(doHideModal()),
-  upload: path => dispatch(doUploadThumbnail(path)),
+  upload: path => dispatch(doUploadThumbnail(path, null, null, fs)),
```

Comment:
same question as above, if `fs` is regularly provided but other params are not, consider moving fs up in param order

when does JS get named parameters?!